cypress/integration/pages/articles/tests.js

```diff
@@ -184,11 +185,132 @@ export const testsThatFollowSmokeTestConfig = ({
           });
         });
       });
+
+      describe.only('Social Embeds', () => {
+        it('Youtube embed is redered when it exists on page', () => {
+          cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
+            const youtubeSocialEmbedsData = body.content.model.blocks.filter(content => content.type === 'social')
```

Comment:
You may be able to use the helper `getBlockData` and `getBlockType` functions for extracting the block data.

Reply:
Seems like those function (`getBlockType`) only return the first occurrence of that block type. 
I will implement new function that return multiple social blocks and select by Provider name.